wdl/src/test/scala/wdl/WdlWorkflowSpec.scala

```diff
@@ -130,14 +131,35 @@ class WdlWorkflowSpec extends WordSpec with Matchers {
       }
     }
 
+    def evaluateOutputs(workflow: WdlWorkflow,
```

Comment:
NB since this is now only used in this test, I've moved it into the test class which used it